dipy/workflows/base.py

```diff
@@ -101,19 +101,33 @@ def add_workflow(self, workflow):
             ref_text = [text if text else "\n" for text in npds['References']]
             ref_idx = self.epilog.find('References: \n') + len('References: \n')
             self.epilog = "{0}{1}\n{2}".format(self.epilog[:ref_idx],
-                                               ''.join([text for text in ref_text]),
+                                        ''.join([text for text in ref_text]),
```

Comment:
Can't you write `''.join(ref_text),`?

Reply:
Hi,
Thanks for the update. I have updated it to ''.join(ref_text).